cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -333,6 +335,14 @@ macro(configure_sdk_unix name architectures)
           message(FATAL_ERROR "unsupported arch for Haiku: ${arch}")
         endif()
         set(SWIFT_SDK_HAIKU_ARCH_x86_64_TRIPLE "x86_64-unknown-haiku")
+      elseif("${prefix}" STREQUAL "WASI")
+        if(NOT arch STREQUAL wasm32)
+          message(FATAL_ERROR "unsupported arch for WASI: ${arch}")
+        endif()
+        set(SWIFT_SDK_WASI_ARCH_wasm32_PATH "${SWIFT_WASI_SDK_PATH}/share/wasi-sysroot")
+        set(SWIFT_SDK_WASI_ARCH_wasm32_TRIPLE "wasm32-unknown-wasi")
+        set(SWIFT_SDK_WASI_ARCH_wasm32_LIBC_INCLUDE_DIRECTORY "${SWIFT_WASI_SDK_PATH}/share/wasi-sysroot/include")
+        set(SWIFT_SDK_WASI_ARCH_wasm32_LIBC_ARCHITECTURE_INCLUDE_DIRECTORY "${SWIFT_WASI_SDK_PATH}/share/wasi-sysroot/include")
```

Comment:
I really would prefer that the we use `SWIFT_WASI_SYSROOT`.  I've been working on getting the WASI libc built with CMake and proper dependencies, and so the output directory changes.  This value would be passed untouched to `SWIFT_SDK_WASI_ARCH_wasm32_PATH`, and then just append `/include` for the LIBC paths.

Reply:
@compnerd I agree that. I updated to use `SWIFT_WASI_SYSROOT` instead of `SWIFT_WASI_SDK_PATH`